.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Is this a stylesheet for your IDE? Should probably git reset and git stash these changes, and then add ./idea/* to .gitignore, unless everyone on the team is using these for IntelliJ.